.github/workflows/ci.yml

```diff
@@ -1,102 +1,72 @@
-
-
 name: Delta Acceptance Testing
 
 on:
-  pull_request:
   push:
-    branches:
-      - main
-
-env:
-      PYTHON_VERSION: 3.9.13
-
+    branches: 
+      -main
+      -fix-ci
+  pull_request:
 
 jobs:
-
-  analyze:
-    name: Lint Code
+  test:
     runs-on: ubuntu-latest
-  
     steps:
-      - name: Checkout repository
-        uses: actions/checkout@v2
-        with:
-          token: ${{ secrets.PAT }}
-
-      - name: Install poetry
-        run: pipx install poetry
+      - uses: actions/checkout@v2
 
-      - name: Set up Python
-        uses: actions/setup-python@v4
+      # If you wanted to use multiple Python versions, you'd have specify a matrix in the job and
+      # reference the matrixe python version here.
+      - uses: actions/setup-python@v2
         with:
-          python-version: ${{ env.PYTHON_VERSION }}
-          cache: 'poetry'          
-
-      - name: Install dependencies and project in dev mode
-        run: |
-          make install
-
-      - name: Lint
-        run: make lint-base lint-mypy
-
-  test-pipeline:
-    name: Run tests and create tables
-    runs-on: ubuntu-latest
-    
-    steps:
-
-      - name: Checkout repository
-        uses: actions/checkout@v2
-
-      - name: Install poetry
-        run: |
-          pipx install poetry
-
-      - name: Set up Python
-        uses: actions/setup-python@v4
+          python-version: 3.9
+
+      # Cache the installation of Poetry itself, e.g. the next step. This prevents the workflow
+      # from installing Poetry every time, which can be slow. Note the use of the Poetry version
+      # number in the cache key, and the "-0" suffix: this allows you to invalidate the cache
+      # manually if/when you want to upgrade Poetry, or if something goes wrong. This could be
+      # mildly cleaner by using an environment variable, but I don't really care.
+      - name: cache poetry install
+        uses: actions/cache@v2
         with:
-          python-version: ${{ env.PYTHON_VERSION }}
-          cache: 'poetry'
-
-      - name: Install dependencies and project in dev mode
-        run: |
-          make install
-
-      - name: Run unit tests
-        run: |
-          make test
-        env:
-          PY_COLORS: 1
-      
-      - name: Reset generated tables
-        if: github.ref == 'refs/heads/main' && github.event_name == 'push'
-        run: make reset-generated-tables
-
-      - name: Write tables
-        run: make write-all
```

Comment:
Are we intentionally stopping writing the tables? Is that intentional?

Reply:
It looks like the code is currently writing the tables, but then not committing or anything, so I'm not really sure what this operation would accomplish.  You'd merge with master, then all the tables would get rewritten, but I don't think anything would be done with the newly generated tables...  I think it's best for the CI to not write the tables so we can try to keep the diffs manageable.